app/src/main/java/org/jellyfin/mobile/model/sql/entity/ServerEntity.kt

```diff
@@ -0,0 +1,25 @@
+package org.jellyfin.mobile.model.sql.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.HOSTNAME
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.TABLE_NAME
+
+@Entity(tableName = TABLE_NAME, indices = [Index(value = arrayOf(HOSTNAME), unique = true)])
+data class ServerEntity(
+    @PrimaryKey(autoGenerate = true)
+    @ColumnInfo(name = SERVER_ID) val id: Long,
+    @ColumnInfo(name = HOSTNAME) val hostname: String,
+    @ColumnInfo(name = LAST_USED_TIMESTAMP) val lastUsedTimestamp: Long,
```

Comment:
Why add the type to the name?

```suggestion
    @ColumnInfo(name = LAST_USED_TIMESTAMP) val lastUsed: Long,
```

Also, "lastActive" might be a better name

Reply:
Dunno, I prefer to state that it's a timestamp.